lib/elixir_boilerplate_web/layouts/templates/app.html.eex

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8"/>
```

Comment:
I think we usually try to keep the space between the last `"` and `/>`? Or did we stop doing that?